### PR TITLE
RDM-12759 - New Global Search FTs

### PR DESCRIPTION
**Pull request by marianadpereira**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-12759

### Change description ###

Global Search Functional tests for pruning of search result set

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by danlysiak — APPROVED** [review]
Scenarios look good! ✅

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -19,7 +19,7 @@ def branchesToSync = ['demo', 'ithc', 'perftest']
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-1159"  // This doesn't change frequently, but when it does, only change this value.
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:22]: This must be reset prior to merger

Comment (marianadpereira) [inline Jenkinsfile_CNP:22]: This must be reset before merging

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -92,7 +92,7 @@ env.DM_STORE_BASE_URL = "http://dm-store-aat.service.core-compute-aat.internal"
 env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```
Comment (marianadpereira) [inline Jenkinsfile_CNP:95]: This must be reset before merging

File: charts/ccd-data-store-api/values.preview.template.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ java:
     DATA_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified"
     DATA_STORE_DB_MAX_POOL_SIZE: 10
-    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-575.service.core-compute-preview.internal/
+    DEFINITION_STORE_HOST: http://ccd-definition-store-api-pr-1159.service.core-compute-preview.internal/
```
Comment (marianadpereira) [inline charts/ccd-data-store-api/values.preview.template.yaml:26]: This must be reset before merging

File: build.gradle [hunk 4]
```diff
@@ -263,7 +264,7 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.9.1'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.10.18-prerelease'
```
Comment (marianadpereira) [inline build.gradle:267]: This needs to be updated to a full release version of the hmcts/ccd-test-defintions before going into master. see https://github.com/hmcts/ccd-test-definitions/pull/32.

**Pull request merged**